tests/users/test_data.py

```diff
@@ -174,3 +174,13 @@ async def test_find_or_create_b2c_user(
         assert int(user.handle.split("-")[-1])
 
     assert user == snapshot(matcher=path_type({"handle": (str,)}))
+
+
+async def test_check_no_users_exist(users_data):
+    """
+    Verifies user existence checks work as expected with and without existing users.
+    """
+    assert await users_data.check_users_exist() is False
+
+    await users_data.create(password="hello_world", handle="bill")
+    assert await users_data.check_users_exist() is True
```

Comment:
Comparison of `bool is True` is redundant since the return value is a bool, aka if it was `True` the comparison returns `True`, If `False` it returns `False`